docs/transaction.md

```diff
@@ -12,24 +12,24 @@ Let's take a look at some very simple transactions:
 ```javascript
 var transaction = new Transaction()
     .from(utxos)          // Feed information about what unspent outputs one can use
-    .to(address, amount)  // Add an output with the given amount of satoshis
+    .to(address, amount)  // Add an output with the given amount of duff
```

Comment:
There are more `satoshis` in comments in other files as well, should probably keep it consistent.